screwdriver.yaml

```diff
@@ -0,0 +1,47 @@
+shared:
+    image: node:8
+jobs:
+    SIMPLE:
+        steps:
+            - echo: echo test
+        requires: [~pr, ~commit]
```

Comment:
Does `SIMPLE` need `~pr`?

Reply:
I would like to use this at:
https://github.com/screwdriver-cd/screwdriver/blob/master/features/workflow.feature#L118